content/scaling-kubernetes-cluster/_index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Scaling Kubernetes Cluster"
```

Comment:
Hrm... thinking `Kubernetes Optimization` since all the other related top-level categories begin with `Kubernetes` (e.g. `Kubernetes Backing Services`, `Kuberneteds Platform Services`)